<ui version="4.0" >
 <author>(C) 2005 Andras Mantia &lt;amantia@kde.org></author>
 <comment>***************************************************************************
 *                                                                         *
 *   This program is free software; you can redistribute it and/or modify  *
 *   it under the terms of the GNU General Public License as published by  *
 *   the Free Software Foundation; either version 2 of the License, or     *
 *   (at your option) any later version.                                   *
 *                                                                         *
 ***************************************************************************
</comment>
 <exportmacro></exportmacro>
 <class>LocalImportWizardPageBase</class>
 <widget class="QWidget" name="LocalImportWizardPageBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>422</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>1</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>620</width>
    <height>410</height>
   </size>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item rowspan="3" row="0" column="0" >
    <widget class="QLabel" name="imagelabel" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>120</width>
       <height>1000</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>85</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addFiles" >
       <property name="text" >
        <string>&amp;Add Files...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addFolder" >
       <property name="text" >
        <string>A&amp;dd Folder...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearList" >
       <property name="text" >
        <string>&amp;Clear List</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>89</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <widget class="SimpleFileTree" name="listView" />
   </item>
   <item row="0" column="1" >
    <widget class="Q3ButtonGroup" name="ButtonGroup12" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Insert existing files</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="checkInsertWithMask" >
        <property name="text" >
         <string>Files with the following &amp;masks:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QLineEdit" name="filterMask" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="whatsThis" >
         <string>List of filtering mask separated by a semicolon. Examples:
&lt;b>*&lt;/b>:  include all files
&lt;b>*.html; *.css&lt;/b>: include files with &lt;i>html&lt;/i> and &lt;i>css&lt;/i> extension</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="KPushButton" name="insertExisting" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>1</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>&amp;Insert</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>340</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="3" >
       <widget class="QRadioButton" name="checkInsertWeb" >
        <property name="text" >
         <string>&amp;Text and image files</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends></extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>SimpleFileTree</class>
   <extends></extends>
   <header>simplefiletree.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends></extends>
   <header>kpushbutton.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>checkInsertWeb</tabstop>
  <tabstop>filterMask</tabstop>
  <tabstop>insertExisting</tabstop>
  <tabstop>addFiles</tabstop>
  <tabstop>addFolder</tabstop>
  <tabstop>clearList</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkInsertWithMask</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterMask</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>251</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel" >
     <x>391</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
